src/windows/TextInput.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+* TextInput.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* RN Windows-specific implementation of the cross-platform TextInput abstraction.
+*/
+
+import React = require('react');
+import RN = require('react-native');
+
+import {applyFocusableComponentMixin, FocusManagerFocusableComponent } from '../native-desktop/utils/FocusManager';
+
+import {TextInput as TextInputBase} from '../native-common/TextInput';
+
+export class TextInput extends TextInputBase implements FocusManagerFocusableComponent {
+
+    protected _render(props: RN.TextInputProps): JSX.Element {
+
+        return (
+            <RN.TextInput
+            {...props}
+            tabIndex = {this.getTabIndex()}
```

Comment:
> =  [](start = 20, length = 3)

no spaces around the `=`

Reply:
ok

---
In reply to: [159358977](https://github.com/Microsoft/reactxp/pull/404#discussion_r159358977) [](ancestors = 159358977)